frontend/src/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, Container, Row, Col } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { getAllowedClasses } from 'helpers/dom/dom';
+import { AppRoute } from 'common/enums/enums';
+import styles from './styles.module.scss';
+
+const NotFound: React.FC = () => {
+  return (
+    <Container
+      className={`${getAllowedClasses(
+        styles.notFoundWrapper,
+      )} d-flex flex-column`}
```

Comment:
You don't need to wrap this in a template literal, `getAllowedClasses` accepts both strings and objects
```suggestion
className={getAllowedClasses(styles.notFoundWrapper), 'd-flex flex-column'}
```